doc/metadata_signing.md

```diff
@@ -0,0 +1,43 @@
+#Proxy Node Metadata Signing
+
+ [Proxy Node metadata](https://proxy-node.eidas.test.london.verify.govsvc.uk/ServiceMetadata) is generated by the [Verify Metadata Controller](https://github.com/alphagov/verify-metadata-controller), from a kubeyaml declaration of
+`kind: Metadata`.
+
+The metadata is signed with a CloudHSM key and its accompanying metadata signing certificate is chained to the following roots of trust in prodution or test:
+
+````
+Production
+├── verify-root-ca-a1
+│   └── proxy-node-metadata-ca-a1
+│       └── verify-eidas-proxy-node-prod-metadata-signing-cert-a1
```

Comment:
Is it always going to be a1?

Reply:
It will still be `a1` until we want to use a new EC keypair. The keypair for `verify-root-ca-a1` for instance can't be deleted with our current codebase.
The certificates signed by these keys can be regenerated when expiring or necessary. If we want a new keypair for reasons, then we'd have to switch the tree to `a2` from the root down.